docs/QUnit/module.md

```diff
@@ -199,10 +199,12 @@ QUnit.test( "makes a car", function( assert ) {
 
 ---
 
-Hooks stack on nested modules
+`before`/`beforeEach` hooks queue on nested modules. `after`/`afterEach` hooks stack on nested modules.
```

Comment:
No, I just missed that. I agree this needs to be reworked later but also agree that consistent messaging now is good.